### PR TITLE
Adds VideoColorSpace

### PR DESCRIPTION
**Pull request by rachelandrew**
Adds `VideoColorSpace` from the WebCodecs spec.

Reviewer: @jpmedley

### CODE REVIEW

File: files/en-us/web/api/videocolorspace/tojson/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  - API
+  - Method
+  - Reference
+  - toJSON
+  - VideoColorSpace
+browser-compat: api.VideoColorSpace.toJSON
+---
+{{DefaultAPISidebar("Web Codecs API")}}
+
+The **`toJSON()`** method of the {{domxref("VideoColorSpace")}} interface is a _serializer_, it returns a JSON representation of the `VideoColorSpace` object.
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/tojson/index.md:14]:
```suggestion
The **`toJSON()`** method of the {{domxref("VideoColorSpace")}} interface is a _serializer_ that returns a JSON representation of the `VideoColorSpace` object.
```

File: files/en-us/web/api/videocolorspace/videocolorspace/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+The **`VideoColorSpace()`** constructor creates a new {{domxref("VideoColorSpace")}} object which represents a video color space.
+
+## Syntax
+
+```js
+new VideoColorSpace();
+new VideoColorSpace(init);
+```
+
+### Parameters
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/videocolorspace/index.md:22]: If `init` is optional, then defaults are used. What are they? They should be identified.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/videocolorspace/videocolorspace/index.md (hunk 2 above)
Comment (rachelandrew) [reply files/en-us/web/api/videocolorspace/videocolorspace/index.md:22]: all null, which is why I didn't list them.

File: files/en-us/web/api/videocolorspace/fullrange/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - Reference
+  - fullRange
+  - VideoColorSpace
+browser-compat: api.VideoColorSpace.fullRange
+---
+{{DefaultAPISidebar("WebCodecs API")}}
+
+The **`fullRange`** read-only property of the {{domxref("VideoColorSpace")}} interface returns `true` if full-range color values are used.
+
+### Value
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/fullrange/index.md:16]:
```suggestion
## Value
```

Comment (jpmedley) [reply files/en-us/web/api/videocolorspace/fullrange/index.md:16]: Apologies. I should have caught this before.

File: files/en-us/web/api/videocolorspace/matrix/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  - Reference
+  - matrix
+  - VideoColorSpace
+browser-compat: api.VideoColorSpace.matrix
+---
+{{DefaultAPISidebar("WebCodecs API")}}
+
+The **`matrix`** read-only property of the {{domxref("VideoColorSpace")}} interface returns the matrix coefficient of the video. Matrix coefficients describe the relationship between sample component values and color coordinates.
+
+### Value
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/matrix/index.md:16]:
```suggestion
## Value
```

File: files/en-us/web/api/videocolorspace/primaries/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  - Reference
+  - primaries
+  - VideoColorSpace
+browser-compat: api.VideoColorSpace.primaries
+---
+{{DefaultAPISidebar("WebCodecs API")}}
+
+The **`primaries`** read-only property of the {{domxref("VideoColorSpace")}} interface returns the color gamut of the video.
+
+### Value
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/primaries/index.md:16]:
```suggestion
## Value
```

File: files/en-us/web/api/videocolorspace/videocolorspace/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
```
Comment (jpmedley) [inline files/en-us/web/api/videocolorspace/videocolorspace/index.md:22]:
```suggestion
### Parameters

All values default to `null` when they are not present.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**